src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/MercuryErrorComponentObject.java

```diff
@@ -0,0 +1,28 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.mercury;
```

Comment:
The same alerts are verified in curated main pages tests in test cases where we are accessing not existing sections/categories and error message is visible to user.
Consider using this component there or create a ticket to unify code base.